#ifndef _AUTOEVENT_PDHEAD_H_ 
#define _AUTOEVENT_PDHEAD_H_
#pragma once
//该文件为使用工具自动生成，请不要在文件中修改，如需要修改，请与sskylin联系。
#include "FriendInfoLite.pb.h"
#include "MsgBuildingItem.pb.h"
#include "MsgMission.pb.h"
#include "MsgPlanet.pb.h"
#include "MsgSocialItems.pb.h"
#include "MsgTransaction.pb.h"
#include "RceCustomizedActivity.pb.h"
#include "RceDeployUnits.pb.h"
#include "RceItemDamaged.pb.h"
#include "RceLogin.pb.h"
#include "RceObtainAskForHelp.pb.h"
#include "RceObtainAttackerList.pb.h"
#include "RceObtainBattleReplay.pb.h"
#include "RceObtainCustomizer.pb.h"
#include "RceObtainHangarsHelp.pb.h"
#include "RceObtainMessagesList.pb.h"
#include "RceObtainNeighborsList.pb.h"
#include "RceObtainNpcList.pb.h"
#include "RceObtainPayUrlParame.pb.h"
#include "RceObtainRandomTargets.pb.h"
#include "RceObtainSocialItems.pb.h"
#include "RceObtainSocialUserInfo.pb.h"
#include "RceObtainUniverse.pb.h"
#include "RceObtainVisitHelps.pb.h"
#include "RceOnlineReward.pb.h"
#include "RceQueryCheckAndLockAccountIfPossible.pb.h"
#include "RceQueryGalaxyWindow.pb.h"
#include "RceQueryReceiveWishItem.pb.h"
#include "RceQueryRefuseReceiveWishItem.pb.h"
#include "RceQuerySendWishItem.pb.h"
#include "RceQueryStarInfo.pb.h"
#include "RceQueryStarsBookmarks.pb.h"
#include "RceQueryVisitHelpsGiftUnitsOnBunker.pb.h"
#include "RceUnitDamaged.pb.h"
#include "RceUpdateAlliances.pb.h"
#include "RceUpdateGameUnits.pb.h"
#include "RceUpdateItem.pb.h"
#include "RceUpdateMisc.pb.h"
#include "RceUpdateMissions.pb.h"
#include "RceUpdateProfile.pb.h"
#include "RceUpdateShips.pb.h"
#include "RceUpdateSocialItem.pb.h"
#include "RceUpdateStarsBookmarks.pb.h"
#include "RceUpdateTargets.pb.h"
#include "RceUpdateVisitHelp.pb.h"
#include "RseAuthState.pb.h"
#include "RseCustomizedActivity.pb.h"
#include "RseDeployUnits.pb.h"
#include "RseItemDamaged.pb.h"
#include "RseLogin.pb.h"
#include "RseObtainAskForHelp.pb.h"
#include "RseObtainAttackerList.pb.h"
#include "RseObtainBattleReplay.pb.h"
#include "RseObtainCustomizer.pb.h"
#include "RseObtainHangarsHelp.pb.h"
#include "RseObtainMessagesList.pb.h"
#include "RseObtainNeighborsList.pb.h"
#include "RseObtainNpcList.pb.h"
#include "RseObtainPayUrlParame.pb.h"
#include "RseObtainRandomTargets.pb.h"
#include "RseObtainSocialItems.pb.h"
#include "RseObtainSocialUserInfo.pb.h"
#include "RseObtainUniverse.pb.h"
#include "RseObtainVisitHelps.pb.h"
#include "RseQueryCheckAndLockAccountIfPossible.pb.h"
#include "RseQueryGalaxyWindow.pb.h"
#include "RseQueryReceiveWishItem.pb.h"
#include "RseQueryRefuseReceiveWishItem.pb.h"
#include "RseQuerySendWishItem.pb.h"
#include "RseQueryStarInfo.pb.h"
#include "RseQueryStarsBookmarks.pb.h"
#include "RseQueryVisitHelpsGiftUnitsOnBunker.pb.h"
#include "RseUnitDamaged.pb.h"
#include "RseUpdateAlliances.pb.h"
#include "RseUpdateGameUnits.pb.h"
#include "RseUpdateItem.pb.h"
#include "RseUpdateMisc.pb.h"
#include "RseUpdateMissions.pb.h"
#include "RseUpdateProfile.pb.h"
#include "RseUpdateShips.pb.h"
#include "RseUpdateSocialItem.pb.h"
#include "RseUpdateStarsBookmarks.pb.h"
#include "RseUpdateTargets.pb.h"
#endif


